models/spleen_deepedit_annotation/configs/multi_gpu_train.json

```diff
@@ -23,14 +23,45 @@
         "shuffle": false
     },
     "validate#dataloader#sampler": "@validate#sampler",
-    "validate#evaluator#val_handlers": "$None if dist.get_rank() > 0 else @validate#handlers",
+    "validate#handlers": [
+        {
+            "_target_": "EarlyStopHandler",
+            "_disabled_": "$not @early_stop",
+            "trainer": null,
+            "patience": 1,
+            "score_function": "$scripts.score_function",
+            "min_delta": 0.01
+        },
+        {
+            "_target_": "StatsHandler",
+            "_disabled_": "$dist.get_rank() > 0",
+            "iteration_log": false
+        },
+        {
+            "_target_": "TensorBoardStatsHandler",
+            "_disabled_": "$dist.get_rank() > 0",
+            "log_dir": "@output_dir",
+            "iteration_log": false
+        },
+        {
+            "_target_": "CheckpointSaver",
+            "_disabled_": "$dist.get_rank() > 0",
+            "save_dir": "@ckpt_dir",
+            "save_dict": {
+                "model": "@network"
+            },
+            "save_key_metric": true,
+            "key_metric_filename": "model.pt"
+        }
+    ],
```

Comment:
I think you don't need to put all the handlers here, can you try:
```json
"validate#evaluator#val_handlers": "$@validate#handlers[: 1 if dist.get_rank() > 0 else None]",
```

Reply:
Hi @Nic-Ma , if not using early stopping, `validate#handlers[: 1]` will be `StatsHandler` thus there will have duplicate logs.
Here the disable way is different between `EarlyStopHandler ` and other handlers, thus I did not use one line code to override. What do you think?

Reply:
Hi @Nic-Ma , in this way when `early_stop` is False, `StatsHandler` will be used on all ranks.

Reply:
If the early_stop is not sure, you can use [: -2] instead of [: 1] to avoid this complicated logic, refer to the train part:
https://github.com/Project-MONAI/model-zoo/blob/dev/models/spleen_deepedit_annotation/configs/multi_gpu_train.json#L18

Thanks.

Reply:
Thanks Nic, it's better to use negative index. I changed to `[: -3]` since we only need the first handler.